recipes/utilities/meta.yaml

```diff
@@ -15,12 +15,13 @@ build:
 
 requirements:
     build:
+        - python
```

Comment:
@rsignell-usgs this was the issue! I somehow forgot the python dependency :flushed:, but I am taking the opportunity to fix some other builds too.